CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.13.0]
+
+- Rework Sandbox API to better support custom Runtime
```

Comment:
I know it's not your doing but `0.12.1` is now released so can we move the headers around so that only `0.13.0` is unreleased?